dags/common/cleanup.py

```diff
@@ -18,7 +20,35 @@ def convert_html_subsripts_to_latex(input):
 def clean_collaboration(input):
     return clean_whitespace_characters(input.replace("for the", ""))
 
+
 def remove_specific_tags(value, tags=None, attributes=None):
     tags = tags or []
     attributes = attributes or []
     return clean(value, tags=tags, attributes=attributes, strip=True)
+
+
+class RemoveLabelTagsContentFilter(Filter):
+    def __iter__(self):
+        label_tag = False
+        for token in Filter.__iter__(self):
```

Comment:
Good question! 
In line 32 we overwriting RemoveLabelTagsContentFilter `__iter__` function (which is built-in func.)  adding label attribute. Then in line 33 ` Filter.__iter__(self)` we iterating per this overwritten RemoveLabelTagsContentFilter class. 
If we will do `super().__iter__()`, we will pass not self of our new class. but self of parent class - so in the end we will overwrite the parent class `__iter__` rather than RemoveLabelTagsContentFilter. Is it clear?

Reply:
Good point @Neodia  and thanks for the explanation @ErnestaP. There is no need to pass `self` when you call a class method for example:


```python

class Something:

    def hello(self):
        print('hello')
       
Something().hello() # <---- you are not passing `self`
```

Reply:
Could you please update it with `super()`?